src/ol/proj/proj4.js

```diff
@@ -19,18 +31,20 @@ import {get as getTransform} from './transforms.js';
  * registry, e.g. after calling `proj4.defs()`. Existing transforms will not be
  * modified by this function.
  *
- * @param {?} proj4 Proj4.
+ * @param {import("proj4")} proj4 Proj4.
```

Comment:
The previous version of jsdoc-plugin-typescript didn't like this syntax.  I've released a new version that does (see https://github.com/openlayers/jsdoc-plugin-typescript/pull/19).